media-sound/pulsemixer/pulsemixer-9999.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+PYTHON_REQ_USE="ncurses"
+
+inherit distutils-r1
+
+DESCRIPTION="cli and curses mixer for pulseaudio"
```

Comment:
Please start with upper case. Since upstreams says "CLI and curses mixer for PulseAudio", you could take that ;-)